updater/lib/dependabot/file_fetcher_job.rb

```diff
@@ -152,20 +152,18 @@ def handle_file_fetcher_error(error)
           }
         when Octokit::Unauthorized
           { "error-type": "octokit_unauthorized" }
+        when Octokit::ServerError
+          # If we get a 500 from GitHub there's very little we can do about it,
+          # and responsibility for fixing it is on them, not us. As a result we
+          # quietly log these as errors
+          { "error-type": "unknown_error" }
         when *Octokit::RATE_LIMITED_ERRORS
-          # If we get a rate-limited error we let dependabot-api handle the
-          # retry by re-enqueing the update job after the reset
```

Comment:
Is removing the above comment intentional?

Reply:
Good catch! Those must've been lost in the shuffle when the splat was moved to the end.